Resync html tests from upstream WPT

cdumez opened the pull request:
#### aaab9f10559694fd7521a2fdc5fe55380433e9ee
<pre>
Resync html tests from upstream WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=243263">https://bugs.webkit.org/show_bug.cgi?id=243263</a>

Reviewed by Tim Nguyen.

Resync html tests from upstream WPT 6473beec7ac068080691.

* LayoutTests/imported/w3c/web-platform-tests/html/*: Updated.

Canonical link: <a href="https://commits.webkit.org/252960@main">https://commits.webkit.org/252960@main</a>
</pre>

nt1m reviewed (approved):
r=me with iOS rebaselined.

Not sure what to do about the WebKit style errors, do we have a blocklist of files to check?

cdumez committed:
Resync html tests from upstream WPT
https://bugs.webkit.org/show_bug.cgi?id=243263

Reviewed by Tim Nguyen.

Resync html tests from upstream WPT 6473beec7ac068080691.

* LayoutTests/imported/w3c/web-platform-tests/html/*: Updated.

Canonical link: https://commits.webkit.org/252960@main
